libeos-payg/provider.c

```diff
@@ -437,7 +437,7 @@ epg_provider_get_code_format_prefix (EpgProvider *self)
 
   EpgProviderInterface *iface = EPG_PROVIDER_GET_IFACE (self);
 
-  g_return_val_if_fail (iface->code_format_prefix == NULL, "");
+  g_return_val_if_fail (iface->code_format_prefix != NULL, "");
```

Comment:
No, this just shouldn't be an assertion.

```c
if (iface->code_format_prefix != NULL)
  return iface->code_format_prefix;

return "";
```

Reply:
Oh, got it